build_deps/tf_dependency/tf_configure.bzl

```diff
@@ -8,6 +8,12 @@ _TF_SHARED_LIBRARY_NAME = "TF_SHARED_LIBRARY_NAME"
 
 _TF_CXX11_ABI_FLAG = "TF_CXX11_ABI_FLAG"
 
+TF_MAJOR_VERSION = "TF_MAJOR_VERSION"
```

Comment:
when cuda is enabled, project name of whl should be 'tensorflow_recommenders_addons_gpu'

Reply:
GPU and CPU version should share the same version number.